db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
```

Comment:
There is ```cinder service-enable``` and ```cinder service-disable``` CLI commands - API calls behind these commands will be needed to implement scale down for storage nodes in the future. Fog currently do not have these implemented but we will need to distinguish Compute Service from Storage Service.